server.js

```diff
@@ -9,7 +9,7 @@ const webpackDevMiddleware = require("webpack-dev-middleware");
 const app = express();
 const compiler = webpack(config);
 
-app.use("/js", express.static("js"));
```

Comment:
I changed it because storybook's server expects it in this format `app.use(express.static("js"))` and based on the express [docs](http://expressjs.com/en/starter/static-files.html) it seemed like the preferred style. I don't think storybook's use case is important enough to warrant the change (we could find another way to load the SVGs, but I would like to take into account express best practices).